src/pyrokinetics/gk_code/cgyro.py

```diff
@@ -1255,6 +1255,7 @@ def _get_coords(
         pos += ntheta_ballooning
 
         ky = grid_data[pos : pos + nky] / bunit_over_b0
+        ky = np.abs(ky)  # Force the sign of ky to be positive
```

Comment:
Could just put the abs around the full statement so we have

`ky = np.abs(grid_data[pos : pos + nky] / bunit_over_b0)`

Reply:
Nah they can just deal with it I think. Don't give them the option I think, they can change it in post as we shouldn't really offer changes in the source code